nixos/modules/programs/qdmr.nix

```diff
@@ -0,0 +1,30 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.qdmr;
+in {
+  meta.maintainers = [ lib.maintainers.janik ];
+
+  ###### interface
+  options = {
+    programs.qdmr = {
+      enable = lib.mkEnableOption (lib.mdDoc ''
+        Whether to configure system to use qdmr.
+        To grant access to a user, it must be part of dialout group:
+        `users.users.alice.extraGroups = [ "dialout" ];`"
```

Comment:
Can we make this part of the module easily?

Reply:
I know that the wireshark module implements that, going to look at that later.

Reply:
Okay, should work now you can compare it with https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/programs/wireshark.nix
I assume that now every user gets added to the dialout group.